docs/REST API/rest-api.md

```diff
@@ -7419,84 +7421,89 @@ Returns a list of global queries or team queries.
+        }
+      ],
+      "stats": {
+        "system_time_p50": 1.32,
+        "system_time_p95": 4.02,
+        "user_time_p50": 3.55,
+        "user_time_p95": 3.00,
+        "total_executions": 3920
       }
-    ],
-    "stats": {
-      "system_time_p50": 1.32,
-      "system_time_p95": 4.02,
-      "user_time_p50": 3.55,
-      "user_time_p95": 3.00,
-      "total_executions": 3920
-    }
-  },
-  {
-    "created_at": "2021-01-19T17:08:24Z",
-    "updated_at": "2021-01-19T17:08:24Z",
-    "id": 3,
-    "name": "osquery_schedule",
-    "description": "Report performance stats for each file in the query schedule.",
-    "query": "select name, interval, executions, output_size, wall_time, (user_time/executions) as avg_user_time, (system_time/executions) as avg_system_time, average_memory, last_executed from osquery_schedule;",
-    "team_id": null,
-    "interval": 3600,
-    "platform": "",
-    "version": "",
-    "automations_enabled": true,
-    "logging": "differential",
-    "saved": true,
-    "observer_can_run": true,
-    "discard_data": true,
-    "author_id": 1,
-    "author_name": "noah",
-    "author_email": "noah@example.com",
-    "packs": [
-      {
-        "created_at": "2021-01-19T17:08:31Z",
-        "updated_at": "2021-01-19T17:08:31Z",
-        "id": 14,
-        "name": "test_pack",
-        "description": "",
-        "platform": "",
-        "disabled": false
+    },
+    {
+      "created_at": "2021-01-19T17:08:24Z",
+      "updated_at": "2021-01-19T17:08:24Z",
+      "id": 3,
+      "name": "osquery_schedule",
+      "description": "Report performance stats for each file in the query schedule.",
+      "query": "select name, interval, executions, output_size, wall_time, (user_time/executions) as avg_user_time, (system_time/executions) as avg_system_time, average_memory, last_executed from osquery_schedule;",
+      "team_id": null,
+      "interval": 3600,
+      "platform": "",
+      "version": "",
+      "automations_enabled": true,
+      "logging": "differential",
+      "saved": true,
+      "observer_can_run": true,
+      "discard_data": true,
+      "author_id": 1,
+      "author_name": "noah",
+      "author_email": "noah@example.com",
+      "packs": [
+        {
+          "created_at": "2021-01-19T17:08:31Z",
+          "updated_at": "2021-01-19T17:08:31Z",
+          "id": 14,
+          "name": "test_pack",
+          "description": "",
+          "platform": "",
+          "disabled": false
+        }
+      ],
+      "stats": {
+        "system_time_p50": null,
+        "system_time_p95": null,
+        "user_time_p50": null,
+        "user_time_p95": null,
+        "total_executions": null
       }
-    ],
-    "stats": {
-      "system_time_p50": null,
-      "system_time_p95": null,
-      "user_time_p50": null,
-      "user_time_p95": null,
-      "total_executions": null
     }
+  ],
+  "meta": {
+    "has_next_results": true,
+    "has_previous_results": false
   }
-]}
```

Comment:
@rachaelshaw I think we'll also want a couple fields related to `count` here, does this look okay?

```
"counts_updated_at": "2024-10-21T18:55:05Z",
"count": 200,

Reply:
This is the latest pattern we're using on newer endpoints as opposed to having a distinct `count` endpoint per entity

Reply:
@jacobshandling making a PR to add this to the docs now. Is `count` typically part of `meta` or is it at the top level?

Reply:
top level